charts/prometheus-node-exporter/values.yaml

```diff
@@ -76,7 +82,8 @@ securityContext:
   runAsNonRoot: true
   runAsUser: 65534
 
-containerSecurityContext: {}
+containerSecurityContext:
```

Comment:
unrelated change. please keep as it is.

Reply:
Srry, format on safe change. :slightly_smiling_face: